assembly/assembly-wsmaster-war/src/main/resources/META-INF/persistence.xml

```diff
@@ -48,6 +48,17 @@
         <class>org.eclipse.che.api.factory.server.model.impl.IdeImpl</class>
         <class>org.eclipse.che.api.factory.server.FactoryImage</class>
 
+        <class>org.eclipse.che.api.workspace.server.model.impl.devfile.ProjectImpl</class>
+        <class>org.eclipse.che.api.workspace.server.model.impl.devfile.ComponentImpl</class>
+        <class>org.eclipse.che.api.workspace.server.model.impl.devfile.EntrypointImpl</class>
+        <class>org.eclipse.che.api.workspace.server.model.impl.devfile.DevfileImpl</class>
```

Comment:
Just a small comment, I would put it at the begging of this section to make it a bit clear that DevfileImpl related objects are listed here

Reply:
\+